lib/features.js

```diff
@@ -103,24 +63,28 @@ module.exports = config => {
     },
 
     /**
-     * Return whether or not this request should bypass sign-in confirmation. Currently,
-     * just checks if user has had a verified security event in the past day.
+     * Return whether or tracking of security history events is enabled.
      *
-     * @param verified
-     * @param recency
      * @returns {boolean}
      */
-    canBypassSiginConfirmation(email, verified, recency) {
-      // If sign-in confirmation is forced for an email, it can't be bypassed.
-      if (signinConfirmation.enabled && signinConfirmation.forcedEmailAddresses.test(email)) {
+    isSecurityHistoryTrackingEnabled() {
+      return securityHistory.enabled
+    },
+
+    /**
+     * Return whether or not we can bypass sign-in confirmation based
+     * on previously seen security event history.
+     *
+     * @returns {boolean}
+     */
+    isSecurityHistoryProfilingEnabled() {
+      if (! securityHistory.enabled) {
         return false
       }
-
-      // IP Profiling returns true if this user has verified a session
-      // within the past day from this ip address.
-      let ipProfilingEnabled = securityHistory.enabled && securityHistory.ipProfiling &&
-        securityHistory.ipProfiling.enabled
-      return ipProfilingEnabled && verified && recency === 'day'
```

Comment:
It seemed awkward to me to have this security-event-history-checking logic in "features.js".  In my head, this file should only be deciding whether various features are enabled or not, rather than containing any of the logic for actually implementing the feature.

Perhaps another way of looking at it: our goal should be for the code in this file to cleanly disappear once the we're finishing developing/testing the feature, and it has "graduated" to full production rollout.